undercloud.yml

```diff
@@ -50,6 +50,7 @@
           --version {{ osp_release }} \
           --build {{ osp_puddle }} \
           --images-task rpm \
+          --hieradata-config "ironic::conductor::default_boot_mode=bios,ironic::drivers::ilo::default_boot_mode=bios" \
```

Comment:
It might be a good idea to add a condition check for `osp_release>17` here, just in case the line interferes with older releases.

Reply:
although this is infra specific, so should be the same for 13, 16 as well but it makes sense to add the release check. i'll add that.